.env_sample

```diff
@@ -1,3 +1,3 @@
-ACCESS_TOKEN=[ENTER_YOUR_TOKEN_HERE]
-APPLICATION_ID=[APPLICATION_ID]
-PUBLIC_KEY=[PUBLIC_KEY]
\ No newline at end of file
+TOKEN=[ENTER_YOUR_TOKEN_HERE]
+DEFAULT_GUILD=953312435337580584
```

Comment:
Why did you put it there again?
The default guild is different for others who want to use this bot!

```suggestion
DEFAULT_GUILD=[ENTER_YOUR_GUILD_ID_HERE]
```

Reply:
Most people will use that one anyways and that is also a great example on how to enter an id

Reply:
Put it into a comment then but with just the id there people could misunderstand this and think they don't have to change anything there
```suggestion
DEFAULT_GUILD=[ENTER_YOUR_GUILD_ID_HERE] # for example 953312435337580584
```

Reply:
```suggestion
DEFAULT_GUILD=953312435337580584 # Guild id of The Botnursery Kindergarten. Change it if you want
```
this better?

Reply:
Not really I prefer my version...

Reply:
@WilsontheWolf @Chicken @BlackDemonFire 
you opinion on default values in the env_sample?

Reply:
Don't matter to me, I personally just leave them empty. Gotta explain them somewhere in markdown anyway.